src/Microsoft.DotNet.TemplateLocator/TemplateLocator.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NuGet.Common;
+using NuGet.Protocol;
+
+namespace Microsoft.DotNet.TemplateLocator
+{
+    public sealed class TemplateLocator
+    {
+        private DirectoryInfo _dotnetSdkTemplatesLocation;
+
+        public TemplateLocator()
+        {
+            string mockTemplateLocation = Environment.GetEnvironmentVariable("MOCKDOTNETSDKTEMPLATESLOCATION");
```

Comment:
Sometimes we put _ in our environment variable names and sometimes we don't.  Is there a reason we choose one or the other?

Reply:
It is for testing. A temporary implementation. We won't need it when it has the proper implementation